src/main/resources/templates/MonitoredTripHtml.ftl

```diff
@@ -0,0 +1,21 @@
+<#include "OtpUserContainer.ftl">
+
+<#--
+    This is a template for an HTML email that gets sent when there are new notifications about an
+    OTP user's monitored trip.
+-->
+
+<#macro email_main>
+    <div>
+        <h2></h2>
```

Comment:
Not sure why there is this empty tag, and we will need to refine which headings use which tags.